tests/sqllogictests/suites/query/functions/02_0077_function_dict_get.test

```diff
@@ -0,0 +1,31 @@
+statement ok
+create or replace table t(a string)
+
+statement ok
+insert into t values('a'),('b'),('c')
+
+statement ok
+CREATE OR REPLACE DICTIONARY d(key string not null, value string not null) PRIMARY KEY key SOURCE(redis(host='127.0.0.1' port='6379'))
+
+query TT
+select dict_get(d, 'value', 'a') from t
```

Comment:
modify this test as `select dict_get(d, 'value', a) from t` to test get value from table columns.